tools/update-configs.ts

```diff
@@ -31,12 +33,21 @@ function formatRules(rules: TCategory['rules']) {
   return JSON.stringify(obj, null, 2)
 }
 
+function formatSingleRule(rules: TCategory['rules'], ruleId: string) {
+  const ruleOpt = rules.find((rule) => rule.ruleId === ruleId)?.meta.docs.recommended || 'error'
+
+  return JSON.stringify({ [ruleId]: ruleOpt }, null, 2)
+}
+
 const SUPPORTED_EXTENSIONS = ['ts', 'tsx', 'js', 'jsx', 'mjs', 'cjs']
 const STORIES_GLOBS = [
   `'*.stories.@(${SUPPORTED_EXTENSIONS.join('|')})'`,
   `'*.story.@(${SUPPORTED_EXTENSIONS.join('|')})'`,
 ]
 
+// Other files that will be linted
+const MAIN_JS_FILE = [`'main.js'`]
```

Comment:
Users can have different extensions for their main file

```suggestion
const MAIN_JS_FILE = [`'main@.(js|cjs|mjs|ts)'`]
```